### PR TITLE
Added a note about proper and common nouns

### PR DESCRIPTION
**Pull request by wordlee**

### CODE REVIEW

File: _pages/capitalization.md [hunk 1]
```diff
@@ -21,11 +21,19 @@ The following are capitalized:
...
+- environment
 - server
 - built-in repository
 - runbooks (the concept not the feature)
 
+## Proper nouns vs common nouns
+
+A proper noun is the name of specific instance of a thing, Australia, Octopus Deploy. A common noun is general concept that refers to a class of things, country, deployment tool. The context words are used in can change whether a word is being used as a proper noun or a common noun. For instance,
+
+> With the Octopus Spaces feature you can create seperate spaces for each of your teams' infrastructure and projects.
```
Comment (kathrynmarks) [inline _pages/capitalization.md:33]: Suggestion: spelling of `separate`

File: _pages/capitalization.md [hunk 2]
```diff
@@ -21,11 +21,19 @@ The following are capitalized:
...
 - deployment target
-- environment 
+- environment
 - server
 - built-in repository
 - runbooks (the concept not the feature)
 
+## Proper nouns vs common nouns
+
+A proper noun is the name of specific instance of a thing, Australia, Octopus Deploy. A common noun is general concept that refers to a class of things, country, deployment tool. The context words are used in can change whether a word is being used as a proper noun or a common noun. For instance,
```
Comment (kathrynmarks) [inline _pages/capitalization.md:31]: Suggestion: 
A proper noun is the name of `a` specific instance of a thing, `such as` Australia `and` Octopus Deploy.

Comment (kathrynmarks) [inline _pages/capitalization.md:31]: Suggestion:
A common noun is `a` general concept that refers to a class of things, such as country `and` deployment tool.

Comment (kathrynmarks) [inline _pages/capitalization.md:31]: Opportunity to remove words.

Suggestion:
The context ~~words are used in~~ can change ...

File: _pages/capitalization.md [hunk 3]
```diff
@@ -21,11 +21,19 @@ The following are capitalized:
...
+
+> With the Octopus Spaces feature you can create seperate spaces for each of your teams' infrastructure and projects.
+
+In the above example, "Octopus Spaces" is the name of the feature, so it's capitalized, however, separate "spaces" refers to spaces as a general class of things and is not capitalized.
```
Comment (kathrynmarks) [inline _pages/capitalization.md:35]: Suggestion: include `separate` in the quotation marks

**Review by kathrynmarks — COMMENTED** [review]
Thank you for adding this to the style guide. I've made a few suggestions that you can take or leave. :)

File: _pages/capitalization.md (hunk 2 above)
Comment (wordlee) [inline _pages/capitalization.md:31]:
```suggestion
A proper noun is the name of a specific instance of a thing, such as Australia and Octopus Deploy. A common noun is a general concept that refers to a class of things, such as country and deployment tool. The context words are used in can change whether a word is being used as a proper noun or a common noun. For instance,
```

File: _pages/capitalization.md (hunk 1 above)
Comment (wordlee) [inline _pages/capitalization.md:33]:
```suggestion
> With the Octopus Spaces feature you can create separate spaces for each of your teams' infrastructure and projects.
```

File: _pages/capitalization.md (hunk 3 above)
Comment (wordlee) [inline _pages/capitalization.md:35]:
```suggestion
In the above example, "Octopus Spaces" is the name of the feature, so it's capitalized, however, "separate spaces" refers to spaces as a general class of things and is not capitalized.
```

**Pull request merged**